src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -1445,6 +1445,42 @@ def setup(self):
             dtype=self.dtype,
         )
 
+    def _pad_to_block_size(
```

Comment:
Thanks for adding this!

Reply:
The main issue still exists when `padding` is not a multiple of `6 * block_size` which is suprising.

I didn't include in this PR since it's BigBird specific and it doesn't seem right to include this hardcoded `6 *` (and it fails other unrelated tests from `tests/test_flax_modeling_bigbird.py` since the seq_length used are NOT mutliples of `6 *`.